### PR TITLE
Enact security policy

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR addresses goals in #1181 
- [ ] Tests for the changes have been added (for bug fixes / features)
  - [x] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [x] CHANGES.rst has been updated (with summary of main changes)
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added

### What kind of change does this PR introduce?

* Adds a security policy

### Does this PR introduce a breaking change?

No.

### Other information:

This Pull Request is waiting on the following:
- [x] Creation of a mail account/redirect specific for the GitHub Organization Admins and major project leads.
    - Done (Office email group redirect for github-support@ouranos.ca) 
- [x] Make this email redirect group public.
- [x] Creation of a PGP key pair for encrypted emails (following email creation).
- [x] `CHANGES.rst` should be updated with this information as an announcement.

This information will be reusable for all projects falling under Ouranosinc.

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@huard @aulemahal

I just want to double-check the PGP key before merging, but otherwise this is ready for review.

**Review by aulemahal — APPROVED** [review]
Héhé I have some difficulties in imagining a "security vulnerability" stemming from xclim (and not from a dependency), but eh. Better safe than sorry.

**Comment by Zeitsperre** [comment]
Same. The way I see this is we now have a security process in place that can serve all of our projects. I can imagine that with PAVICS-related projects, server-side security is a much harder thing to secure. You never know when an issue could come up there.

This also nearly completes the OpenSSF certification, so that's also pretty cool.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Security Policy
+
+[//]: # (ATTENTION: This is the Markdown version of docs/security.rst. Any changes should also be made in the ReStructuredText version.)
+
```
Comment (huard) [inline SECURITY.md:4]: Why do we need two versions ?

**Review by huard — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (Zeitsperre) [reply SECURITY.md:4]: The convention for security notices is to have them in Markdown, but I couldn't find an easy way to load Markdown into Sphinx (without adding another library to the docs dependencies, like [sphinx-mdinclude](https://github.com/omnilib/sphinx-mdinclude)).

What do you think? Is it worth adding it in?

Comment (Zeitsperre) [reply SECURITY.md]: Done.

**Pull request merged**